src/globals/versions.ts

```diff
@@ -0,0 +1,10 @@
+import * as semver from "semver";
+import tabnineExtensionProperties from "./tabnineExtensionProperties";
+const AUTHENTICATION_API_VERSION = "1.54.0";
+
+export default function isAuthenticationApiSupported(): boolean {
```

Comment:
You can move the "minimal supported version" check we have in the pre release installer to here too